Dockerfile

```diff
@@ -1,40 +1,27 @@
-FROM debian:jessie
-
-RUN apt-get update
-RUN apt-get install -y apt-transport-https
-RUN apt-get install -y ca-certificates
-RUN apt-get install -y bsdiff git curl
```

Comment:
bsdiff is required by the tests - is it required by the main program as well?  Most of this stuff was no longer needed since we don't need to install Go anymore.